### DISCUSSION TITLE
Debug windows python

### DISCUSSION THREAD

**Discussion by koren-gershoni** [ayamir/nvimdots#1204] [General]
Hi, I open here because I don't know if it is a bug or not.
I am trying to debug python. I put breakpoints in the code, I press F6
it shows me the following window:
![image](https://github.com/ayamir/nvimdots/assets/57925934/a780bcee-df36-4fda-bd68-1640d29d53b1)
I press 2 (tried also 1)
then it shows me the following window with the path of the file:
![image](https://github.com/ayamir/nvimdots/assets/57925934/dbfb6045-cb6b-44c0-983b-67adb7ceb7a6)
I press enter and then it opens another window of cmd/shell with the following:
![image](https://github.com/ayamir/nvimdots/assets/57925934/ab528d2b-1046-4bba-82b4-7247bff2842f)

what am I missing?

**Comment by charliie-dev** [comment]
seems normal to me, it looks like u placed breakpoints at "import"?

**Reply by koren-gershoni** [reply]
I didn't place a breakpoint at import.
it doesn't let me to actually debug, its open that new window and that's all, nothing in the original screen, can't see the ui.
if I press F6 again I can see the following window:
![image](https://github.com/ayamir/nvimdots/assets/57925934/4ef2a086-734d-4a58-8886-8b6821feb5e1)
and it never stops (not sure if it runs at all) on any line with a break point.

**Comment by ayamir** [comment]
Seems `nvim-dap-ui` requires `nvim-neotest/nvim-nio` but our current configuration has not yet added it to the dependencies.

**Comment by ayamir** [comment]
OK, it can be reproduced stableily on windows:
![image](https://github.com/ayamir/nvimdots/assets/61657399/76cd5b35-07b7-4996-a4cb-f8eedee319be)

**Comment by ayamir** [comment]
https://www.reddit.com/r/neovim/comments/199jt2c/nvimdap_debugpy_cant_work_in_windows/

**Comment by ayamir** [comment]
https://github.com/ayamir/nvimdots/pull/1205
